Sources/ClientRuntime/Retries/RetryErrorType.swift

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+import AwsCommonRuntimeKit
+
+/// A classification that is assigned to an error returned by an operation, for the purpose of
+/// determining whether & how the operation should be retried.
+///
+/// > Note: Based on _The Design and Implementation of the AWS SDKs_ section 3.2.1, The Smithy Retry Interface, `RetryErrorType`.
```

Comment:
If this document isn't public, then we shouldn't include mentions of it in public comments